### PR TITLE
Describe (a,) vs (a) in docs

### LINKED ISSUE #24730: Reference should mention tuple/parenthesis disambiguation
Using (a,) to disambiguate from (a).

### PR DESCRIPTION
**Pull request by steveklabnik**
Fixes #24730 

r? @alexcrichton

### CODE REVIEW

File: src/doc/reference.md [hunk 1]
```diff
@@ -2431,11 +2431,18 @@ Tuples are written by enclosing zero or more comma-separated expressions in
...
-(0,);
 (0.0, 4.5);
 ("a", 4usize, true);
 ```
 
+You can disambiguate a single-element tuple from a value in parenteses with a
+comma:
+
+```
+(0,); // sigle element tuple
```
Comment (parir) [inline src/doc/reference.md]: s/sigle/single/

File: src/doc/trpl/primitive-types.md [hunk 2]
```diff
@@ -248,6 +248,14 @@ or “breaks up” the tuple, and assigns the bits to three bindings.
 
 This pattern is very powerful, and we’ll see it repeated more later.
 
+You can disambiguate a single-element tuple from a value in parenteses with a
+comma:
+
+```
+(0,); // sigle element tuple
```
Comment (parir) [inline src/doc/trpl/primitive-types.md]: s/sigle/single/

File: src/doc/reference.md [hunk 3]
```diff
@@ -2431,11 +2431,18 @@ Tuples are written by enclosing zero or more comma-separated expressions in
 parentheses. They are used to create [tuple-typed](#tuple-types) values.
 
 ```{.tuple}
-(0,);
 (0.0, 4.5);
 ("a", 4usize, true);
 ```
 
+You can disambiguate a single-element tuple from a value in parenteses with a
```
Comment (parir) [inline src/doc/reference.md]: s/parenteses/parentheses/

File: src/doc/trpl/primitive-types.md [hunk 4]
```diff
@@ -248,6 +248,14 @@ or “breaks up” the tuple, and assigns the bits to three bindings.
 
 This pattern is very powerful, and we’ll see it repeated more later.
 
+You can disambiguate a single-element tuple from a value in parenteses with a
```
Comment (parir) [inline src/doc/trpl/primitive-types.md]: s/parenteses/parentheses/

**Comment by steveklabnik** [comment]
Thanks @parir , sigh.

File: src/doc/reference.md [hunk 5]
```diff
@@ -2431,11 +2431,18 @@ Tuples are written by enclosing zero or more comma-separated expressions in
...
-(0,);
 (0.0, 4.5);
 ("a", 4usize, true);
 ```
 
+You can disambiguate a single-element tuple from a value in parentheses with a
+comma:
+
+```
+(0,); // single element tuple
```
Comment (apasel422) [inline src/doc/reference.md]: s/single element/single-element/

File: src/doc/trpl/primitive-types.md [hunk 6]
```diff
@@ -248,6 +248,14 @@ or “breaks up” the tuple, and assigns the bits to three bindings.
 
 This pattern is very powerful, and we’ll see it repeated more later.
 
+You can disambiguate a single-element tuple from a value in parentheses with a
+comma:
+
+```
+(0,); // single element tuple
```
Comment (apasel422) [inline src/doc/trpl/primitive-types.md]: Same here.

File: src/doc/reference.md (hunk 5 above)
Comment (steveklabnik) [inline src/doc/reference.md]: you're always good at catching those :)

**Pull request merged**